### PR TITLE
release: 0.4

### PR DESCRIPTION
**Pull request by austince**
Upgraded to support Flink 1.5. Not sure what your release workflow is like but if this looks good I'll update for releases 0.5 and 0.6 for Flink 1.6 and 1.7, respectively.

### CODE REVIEW

**Comment by knaufk** [comment]
Thanks @austince and sorry for not getting back earlier. I'l lhave a look at it over the holidays and prepare a new release.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ It builds upon Flink's `TestEnvironment` and `TestStreamEnvironment` and can be
 
 | Version |  Flink Version | Java Version |
 | --------------- | ------------- | ------------ |
+| 0.4    | 1.5.0  | 1.7 |
```
Comment (knaufk) [inline README.md:20]: If we are planning to support further version of Flink we should remove the note above the compatibility matrix

**Review by knaufk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (austince) [reply README.md:20]: Why? Won't it still be important for users of this library to know how to match versions to Flink?

Comment (knaufk) [reply README.md:20]: We need to keep the table, yes. Currently I have a warning above the table saying that I will not upgrade this library to new Flink version, which we can probably remove now. Anyway, I will just merge and add the commit myself.

**Review by knaufk — APPROVED** [review]

**Pull request merged**